cfme/automate/buttons.py

```diff
@@ -437,9 +436,14 @@ class NewButtonGroupView(ButtonGroupFormCommon):
 
     @property
     def is_displayed(self):
+        if self.browser.appliance.version < "5.10":
+            expected_title = "Adding a new Buttons Group"
+        else:
+            expected_title = "Adding a new Button Group"
```

Comment:
It seems like a place where `VersionPicker` should be used.

Reply:
@jan-zmeskal personally I avoid using `version pick` in a view definition; where I can easily access version attribute of an appliance.